src/components/sidebar/splitLinkView/SplitLinkInfo.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { ILink, INode } from '~/models';
+import TransitTypeHelper from '~/util/TransitTypeHelper';
+import NodeHelper from '~/util/nodeHelper';
+import NodeType from '~/enums/nodeType';
+import TextContainer from '../TextContainer';
+import * as s from './splitLinkInfo.scss';
+
+interface ISplitLinkInfoProps {
+    link: ILink;
+    node: INode;
+}
+
+const SplitLinkInfo = (props: ISplitLinkInfoProps) => (
+    <div className={s.splitLinkInfo}>
+        <div className={s.formSection}>
+            <div className={s.sectionHeader}>
+                Jaettava linkki
+            </div>
+            <div className={s.flexRow}>
+                <TextContainer
+                    label='ALKUSOLMU'
+                    value={props.link.startNode.id}
+                />
+                <TextContainer
+                    label='LOPPUSOLMU'
+                    value={props.link.endNode.id}
+                />
+            </div>
+            <div className={s.flexRow}>
+                <TextContainer
+                    label='VERKKO'
+                    value={TransitTypeHelper.getTransitTypeLabel(props.link.transitType!)}
+                />
```

Comment:
Just an idea:
```
{TransitTypeHelper.getTransitTypeIcon(props.link.transitType!}
<TextContainer
    label='VERKKO'
    value={TransitTypeHelper.getTransitTypeLabel(props.link.transitType!)}
/>
```

Reply:
Maybe, can try

Reply:
Yes indeed, it could be done at #831. Added this note there